### DISCUSSION TITLE
Update to french locale [fr]

### DISCUSSION THREAD

**Discussion by RoueSoify** [aclap-dev/video-downloadhelper#467] [Language & Translation]
[fr.txt](https://github.com/aclap-dev/video-downloadhelper/files/15248804/fr.txt)
Couldn't attach a .toml, so made it a .txt

**Comment by paulrouget** [comment]
> Couldn't attach a .toml

Oh, can you explain?

**Reply by paulrouget** [reply]
ok - you're correct. I'll fix it.

**Comment by paulrouget** [comment]
And thank you for the traduction!

**Comment by mrosseni** [comment]
Pour combien de temps la licence est-elle valide ?

**Reply by paulrouget** [reply]
Pour toujours.

**Comment by thelwyn** [comment]
Plus de chaines traduites en fr.
[fr.txt](https://github.com/aclap-dev/video-downloadhelper/files/15269949/fr.txt)

**Comment by paulrouget** [comment]
> +v9_settings_button_import = "Paramètres d'importation"
> +v9_settings_button_export = "Paramètres d'exportation"

ça devrait etre "Export/Importer les paramètres".

Peux tu refaire une mise a jour? Merci :)

**Reply by thelwyn** [reply]
Voilou
[fr.txt](https://github.com/aclap-dev/video-downloadhelper/files/15281098/fr.txt)

**Comment by paulrouget** [comment]
Merci - ce sera fixé dans 9.0.1.10.
